lms/templates/certificates/_accomplishment-rendering.html

```diff
@@ -51,7 +51,7 @@ <h3 class="accomplishment-signatories-title sr-only">${_("Noted by")}</h3>
 
                                     <h4 class="signatory-name hd-5">${signatory['name']}</h4>
                                     <p class="signatory-credentials copy copy-micro">
-                                        <span class="role">${signatory['title']|linebreaks}</span>
```

Comment:
You should be good to remove the linebreaks import because L54 is the only spot we're using it in this template.